app/services/search/google_search_service.rb

```diff
@@ -4,18 +4,26 @@ module Search
   class GoogleSearchService
     GOOGLE_SEARCH_URL = 'https://www.google.com/search?q=%{keyword}&hl=%{language}&lr=%{language}'
 
+    USER_AGENT_HEADER = 'User-Agent'
+
     def initialize(keyword, language = 'en')
       @search_url = format(GOOGLE_SEARCH_URL, keyword: keyword, language: language)
     end
 
     def search
       uri = URI.parse(@search_url)
 
-      response = Faraday.get(uri)
+      response = Faraday.get(uri, nil, USER_AGENT_HEADER => generate_user_agent)
 
-      response.body if response.status == 200
+      @html_code = response.body if response.status == 200
```

Comment:
Is this change necessary? If I'm not mistaken we can omit `@html_code`

Reply:
@Tuubz Good catch 🙈